Automattic/wp-calypso
# Modularise posts selectors

This is a reorganisation of posts selectors, which all lived inside a single `selectors.js` file. This PR splits them into individual modules, re-exporting them from an index file. It's a follow-up to the posts state modularisation that took place in #39625.

I split each extraction into its own commit, in case that makes things easier for reviewers.

This PR *may* help with loading performance, in case there are any seldom-used albeit heavy selectors in there, as they can now move freely to just the chunks they're needed in.

#### Changes proposed in this Pull Request

* Split `state/posts/selectors.js` into individual modules

#### Testing instructions

This PR only changes where things live, with no changes to functionality. As such, it should be sufficient to ensure that the build continues to work and the existing unit tests don't fail.
